mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -239,15 +249,30 @@ func invokeDeleteCRDs() error {
 	cfg := controllerConfig.GetConfigOrDie()
 	client, err := apiextension.NewForConfig(cfg)
 	if err != nil {
-		return errors.Wrap(err, "cannot invoke delete CRDs")
+		err = errors.Wrap(err, "cannot invoke delete CRDs")
+		if !forceDelete {
+			return err
+		}
+
+		log.Debug(err)
 	}
 
 	if err = deleteCRD(brokderCRDName, client); err != nil {
-		return errors.Wrap(err, "cannot delete CRD "+brokderCRDName)
+		err = errors.Wrap(err, "cannot delete CRD "+brokderCRDName)
+		if !forceDelete {
+			return err
+		}
+
+		log.Debug(err)
 	}
 
 	if err = deleteCRD(meshsyncCRDName, client); err != nil {
-		return errors.Wrap(err, "cannot delete CRD "+meshsyncCRDName)
+		err = errors.Wrap(err, "cannot delete CRD "+meshsyncCRDName)
+		if !forceDelete {
+			return err
```

Comment:
@gr455, are you familiar with this framework? https://docs.meshery.io/project/contributing/contributing-error

Reply:
@leecalcote Thanks for sharing, I gave it a read but is there something in particular you would like me to consider?

Reply:
@gr455, yes, using the framework (MeshKit errors) when catching and cataloging errors.

Reply:
Got it. But do my changes seem to violate any of those conventions? I'm sorry I cannot tell :sweat_smile:

Reply:
He's asking you to replace these errors to Meshkit errors

Reply:
@gr455 can you try investigating a question that I have? 
Why doesn't `invokeDeleteCRDs` fail when broker and meshsync haven't deployed yet and I run `stop` command??
It should return the same error `"meshery-broker" not found`

Reply:
I'm not sure. I tried to forcefully delete a CRD, and it does error out as expected
```
cannot delete CR meshery-broker: the server could not find the requested resource
cannot delete CRD brokers.meshery.layer5.io: customresourcedefinitions.apiextensions.k8s.io "brokers.meshery.layer5.io" not found
```